### DISCUSSION TITLE
Is there a way to "delete all" entries or select and delete multiple entries?

### DISCUSSION THREAD

**Discussion by mdkberry** [Athou/commafeed#1068] [Q&A] [answered]
I was re-importing the exported OPML, and this works for RSS feeds that don't required settings in the Feed URL.
but when re-import an OPML and the existing entries have changed in the Feed URL then it creates a new entry (effectively a duplicate with different settings).

example:
in the xml I have this for one entry:

```
<outline text="Twitter" title="Twitter">
      <outline text="Twitter @Username" type="rss" title="Twitter @Username" xmlUrl="https://rsshub.app/twitter/user/username/readable=1&amp;authorNameBold=1&amp;showQuotedAuthorAvatarInDesc=1&amp;showEmojiForRetweetAndReply=1&amp;addLinkForPics=1&amp;showTimestampInDescription=1&amp;showQuotedInTitle=1&amp;heightOfPics=150" htmlUrl="https://twitter.com/username" />
```

If I change the above in the OPML xml file and then re-import it wont overwrite the original it will add a new one. Which is fine, but because I can see no way to delete multiple entries I then have to delete the originals one by one.

**Comment by Athou** [comment, marked as answer]
You're right, the OPML importer checks if a feed exists by comparing the URL, not the name you gave it in the "title" attribute. If you change the URL each time you import your OPML file, you will have duplicates. This is by design. 

There's currently no way to mass edit/delete subscriptions :/

**Comment by mdkberry** [comment]
as a work around to this - I am on Windows using Github repo version - is there a way to delete a profile cleanly, so I dont have to rebuild the Github or one-by-one delete all the entries to bring it to the original state?

**Reply by Athou** [reply]
If you're using the default database settings, you can delete the database, it will be recreated with the default "admin" user the next time commafeed starts. You need to delete those two files:
- commafeed-server/target/example.mv.db
- commafeed-server/target/example.trace.db

**Reply by mdkberry** [reply]
can't seem to find those on the windows install. nothing in a search, and in the \sessions folder there is just a single file with no extension

**Reply by mdkberry** [reply]
it's okay. I found them under `C:\commafeed\data`, not in the github repository location
